p2p/peer.py

```diff
@@ -332,9 +332,15 @@ def is_closing(self) -> bool:
         while True:
             try:
                 cmd, msg = await self.read_msg()
-            except (PeerConnectionLost, TimeoutError) as e:
+            except (PeerConnectionLost, TimeoutError) as err:
                 self.logger.debug(
-                    "%s stopped responding (%s), disconnecting", self.remote, repr(e))
+                    "%s stopped responding (%r), disconnecting", self.remote, err)
+                return
+            except DecryptionError as err:
+                self.logger.debug(
+                    "Unable to decrypt message from %s, disconnecting: %r",
```

Comment:
Hm, any way this could be a spoofed message from a different peer? Just thinking that if a malicious peer could spoof messages from other peers (even with broken encryption), and cause us to drop all other peers.

Reply:
Possibly, I was planning on maybe bumping this to `WARN` level for a while to see if it shows up again so we don't loose it in our debug logs.

Reply:
Upped it to `WARN` level and included a stack-trace in the logging output for good measure.